static_src/stores/app_store.js

```diff
@@ -18,23 +16,42 @@ class AppStore extends BaseStore {
   }
 
   _registerToActions(action) {
-    switch(action.type) {
+    switch (action.type) {
       case appActionTypes.APP_FETCH:
         cfApi.fetchApp(action.appGuid);
         break;
 
+      case appActionTypes.APP_STATS_FETCH:
+        cfApi.fetchAppStats(action.appGuid);
+        break;
+
       case appActionTypes.APP_RECEIVED:
         // TODO only emit change event if updates actually change local data.
         this._data = this._merge(this._data, [action.app]);
         this.emitChange();
         break;
 
+      case appActionTypes.APP_STATS_RECEIVED: {
+        const idx = this._data.findIndex((app) => app.guid === action.appGuid);
+        if (idx < 0) {
+          let newApp = {
+            guid: action.appGuid
+          };
+          newApp = Object.assign({}, newApp, action.app);
+          this._data.push(newApp);
+        } else {
+          this._data[idx] = Object.assign({}, this._data[idx], action.app);
```

Comment:
I don't think this is the preferred way once #256 gets merged. It would be better to use the `.push()` or `.merge()` methods